src/status_im/contexts/wallet/send/flow_config.cljs

```diff
@@ -1,11 +1,13 @@
-(ns status-im.contexts.wallet.send.flow-config)
+(ns status-im.contexts.wallet.send.flow-config
+  (:require
+    [status-im.contexts.wallet.send.utils :as send-utils]))
 
 (defn- collectible-selected?
   [db]
   (let [collectible-stored (-> db :wallet :ui :send :collectible)
         tx-type            (-> db :wallet :ui :send :tx-type)]
     (and (some? collectible-stored)
-         (= tx-type :collectible))))
+         (send-utils/tx-type-collectible? tx-type))))
```

Comment:
Why are you checking for collectible tx type that way.. you seem to be checking for
```
(def ^:private collectible-tx-set
  #{:tx/collectible-erc-721
    :tx/collectible-erc-1155})
```
if it contains `:collectible`, what is the benefit of that over `(= tx-type :collectible)` ?

Reply:
The `tx-type` used to be just `:collectible` before because we supported only `erc-721` collectibles. But now it is either `:tx/collectible-erc-721` or `:tx/collectible-erc-1155`. The `:collectible` keyword has been renamed to `:tx/collectible-erc-721`.

So a simple `=` for `:collectible` would not suffice, as we have to check if the `tx-type` is either of the 2 supported collectible types.

cc: @ulisesmac

Reply:
Totally makes sense for me now. 
Thanks @BalogunofAfrica for the explanation, btw, I think the new name for the function is better :+1: